doc/api/util.md

```diff
@@ -1810,13 +1810,24 @@ console.log(util.stripVTControlCharacters('\u001B[4mvalue\u001B[0m'));
 added:
   - v21.7.0
   - v20.12.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/54389
+    description: Respect isTTY and colors environment variables
+      such as NO_COLORS, NODE_DISABLE_COLORS, and FORCE_COLOR.
 -->
 
 * `format` {string | Array} A text format or an Array
   of text formats defined in `util.inspect.colors`.
 * `text` {string} The text to to be formatted.
+* `options` {Object}
```

Comment:
You need to add this to the function's signature above.

Reply:
Done in [cc08b60](https://github.com/nodejs/node/pull/54389/commits/cc08b60053766f5224d2aa95fdc27135e1933412)

Reply:
It's not what I meant. You have to update https://github.com/nodejs/node/blob/22daeba24d0dacc2d9377d9c46c562da27260f9e/doc/api/util.md#L1805

Reply:
@RafaelGSS I've updated the doc for ya :)